<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetThumbnail</class>
 <widget class="QWidget" name="WidgetThumbnail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ThumbnailLabel" name="thumbnailLabel_1">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ThumbnailLabel" name="thumbnailLabel_2">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ThumbnailLabel" name="thumbnailLabel_3">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ThumbnailLabel" name="thumbnailLabel_4">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ThumbnailLabel" name="thumbnailLabel_5">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ThumbnailLabel" name="thumbnailLabel_6">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ThumbnailLabel" name="thumbnailLabel_7">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ThumbnailLabel" name="thumbnailLabel_8">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ThumbnailLabel</class>
   <extends>QLabel</extends>
   <header>thumbnaillabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
